modules/ddl-and-loading/pages/attribute-data-types.adoc

```diff
@@ -120,6 +120,10 @@ Existing schemas that are using `STRING COMPRESS` can continue to function norma
 If you need reference for `STRING COMPRESS`, please refer to GSQL Language Reference version 3.5 or older.
 ====
 
+[NOTE]
+====
+Primitive data types used as attributes of a vertex or edge, can be set to `NULL`.
+====
```

Comment:
I would make this an entire (brief) section on Null Values
* If an attribute is defined as NULLABLE, then its value can be NULL, in addition to its characteristic values
* what does it signify if an attibute is null?
* how is the null value indicated as in input?